Sources/Support/PaywallExtensions.swift

```diff
@@ -124,8 +124,8 @@ private extension View {
                     Logger.appleWarning(Strings.configure.sk2_required_for_swiftui_paywalls)
                 }
 
-                Purchases.shared.cachePresentedOfferingIdentifier(
-                    offering.identifier,
+                Purchases.shared.cachePresentedOfferingContext(
+                    PresentedOfferingContext(offeringIdentifier: offering.identifier),
```

Comment:
Nit: I like to do this to avoid all the duplicated "offering context"
```suggestion
                    .init(offeringIdentifier: offering.identifier),
```